fluent-dom/src/localization.js

```diff
@@ -55,17 +55,49 @@ export default class Localization {
    */
   async formatWithFallback(keys, method) {
     const translations = [];
+    let hasMissing = true;
+    let debugInfo = null;
+
     for (let ctx of this.ctxs) {
       // This can operate on synchronous and asynchronous
       // contexts coming from the iterator.
       if (typeof ctx.then === "function") {
         ctx = await ctx;
       }
-      const errors = keysFromContext(method, ctx, keys, translations);
-      if (!errors) {
+      if (keysFromContext(method, ctx, keys, translations)) {
+        hasMissing = false;
```

Comment:
This will only set `hasMissing` to `false` if the current context has _all_ the translations. However, if the sum of two or more contexts is also complete but none of the context is complete individually, `hasMissing` will still be `true`.